### ISSUE TITLE
whitelist fastly.net, it's a CDN and without it many site are broken such as coopvoce.it

### ISSUE THREAD

**Issue by lucamp** [EnergizedProtection/block#914]
<h1 align="center">Energized Protection - block ⚡</h1>
<div align="center">
  <strong>Let's make an annoyance free, better open internet, altogether!</strong>
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following info properly, which will help us to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [x] Whitelist
- [ ] Blacklist
- [ ] App Broken
- [ ] Website Broken
- [ ] Request
- [ ] Other Issue

<hr>

### Pack(s):
Write the name of the pack(s) you are using.

- Pack(s) Name: Basic

<hr>

### Extension(s):
Also name the extension(s) if you are using any. If you don't then leave it blank.

- Extension(s) Name: 

<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send us the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
*.fastly.net
```

<hr>

### Your Config:
Just to ensure there is no issues or conflicts with other app/software/magisk module/extension/source list. 
Make sure you are running Energized Protection Service only.

- Client: ublock

- Version: 22.05.129

<hr>

### Detail:
Write us a lil bit more about your issue or query. You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail: 

fastly.net is a CDN
sites such as coopvoce.it do not work without it, in this single case to manually fix only this one I added this on my filters
```
@@||adobe-aem.map.fastly.net^$domain=www.coopvoce.it
```

<hr>

<div align="center">
  <h2>Thank you for making Energized Protection great, with your kind help!</h2>
  <sub>A project by <a href="https://nayemador.com" target="_blank">Ador</a> with ❤<pub>
</div>

**Comment by DDL56x** [comment]
Be that as it may, the following is a search engine hijacker and I have blocked it at my OS (Win 11 Home)  firewall:

151.101.121.194 http://www.goto-searchitnow.global.ssl.fastly.net

**Comment by DDL56x** [comment]
Was search jacked late last year while using the browser Vivaldi, but it was on a fully security patched Win 7 Ultimate x64 OS. The browser became completely compromised, so I uninstalled  it using Revo in safe mode. Noticed that more and more websites  since then are or have Fastly as a CDN, and was viewing  a Chrome extension homepage with uBO, Ghostery. and Malwarebytes active when it occurred. I have scoured the web, but found very little info about the malware anywhere. Maybe fastly(dot)net has shut them down, but until I know for sure,  tend to avoid whitelisting that CDN unless necessary.
